src/test/java/org/broadinstitute/hellbender/utils/gcs/BucketUnitUtilsTest.java

```diff
@@ -202,4 +244,28 @@ public void testGetWrapper(int bufferSize, boolean prefetchingIsEnabled) throws
             Assert.assertEquals(wrapped instanceof SeekableByteChannelPrefetcher, prefetchingIsEnabled);
         }
     }
+
+    @Test(groups="cloud")
+    public void testCreateSignedUrl() throws IOException {
+        final String gcsPathString = getGCPTestInputPath() + "nio/big.txt";
+        Assert.assertTrue(Files.exists(IOUtils.getPath(gcsPathString)), "test file is missing, " + gcsPathString);
+
+        final String signed = BucketUtils.createSignedUrlToGcsObject(gcsPathString, 1);
+        final Path path = IOUtils.getPath(signed);
+        Assert.assertTrue(signed.startsWith("https://"), "path doesn't start with https, " + signed);
+        Assert.assertTrue(signed.contains("big.txt"), "path is missing blob name, "+ signed);
+        Assert.assertTrue(Files.exists(path), "path doesn't exist: " + signed);
+        try(final Stream<String> lines = Files.lines(path))  {
+            Assert.assertTrue(lines.anyMatch(line -> line.contains("The Project Gutenberg EBook of The Adventures of Sherlock Holmes")), "blob data is incorrect, " + signed);
```

Comment:
Just checking that the runtime of this test is reasonable (since the file is named "big.txt")?

Reply:
yeah, that's the first line